### PR TITLE
Add meteomatics weather map docs.

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Add documentation for Meteomtaics WMS based weather map.
Need to update the query and their responses and also add the correct link!

### Checklist

- [x] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [x] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+   :description: UP42 data blocks: Weathet/Ocean data forecasts block
+   :keywords: weather, ocean, forecast, meteomatics, model data,
+              observational data, time series
+
+.. _meteomatics-weather-map-block:
+
+Meteomatics - Weather/Ocean Data and Forecasts Map
+================================
+
+For more information, please read the ...
```
Comment (markusuwe) [inline source/up42-blocks/data/meteomatics-weather-map.rst:11]: Read the what?

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+to the default variables list according to description provided by
+Meteomatics. For more information about other variables please refer
+to `Meteomatics website
+<https://www.meteomatics.com/en/api/available-parameters/basic-weather-parameter/>`_
+For more information on supported filters, see :ref:`query filter
+section <filters>`.
+
+* ``bbox``: The bounding box to use as an area of interest (AOI). Will return all scenes that intersect with this box. Use only ``bbox``
+  **or** ``intersects`` **or** ``contains``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use only ``bbox``
```
Comment (markusuwe) [inline source/up42-blocks/data/meteomatics-weather-map.rst:35]: So, is contains supported or not?

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+  In this block we select the ``mix`` option provided by Meteomatics
+  which combines different models and sources into an *intelligent*
+  blend, such that the best data source is chosen for each time and
+  location. The length of the forecasting period as well as the
+  spatial resolution depends on the model from which the requested
+  parameters originate.
+
+.. warning::
+
+  In this block you can choose ``pixel_resolution`` to be a number from 2
```
Comment (markusuwe) [inline source/up42-blocks/data/meteomatics-weather-map.rst:54]: Nice!

**Review by markusuwe — COMMENTED** [review]
LGTM

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 1 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:11]: Yes, this what I put as a cm in my PR, I will add this later

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 2 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:35]: Oh, good catch, I need to remove contain

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 1 above)
Comment (markusuwe) [reply source/up42-blocks/data/meteomatics-weather-map.rst:11]: Oh right, should have read more carefully.

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 4]
```diff
@@ -0,0 +1,301 @@
+.. meta::
+   :description: UP42 data blocks: Weathet/Ocean data forecasts block
```
Comment (thaisbendixen42) [inline source/up42-blocks/data/meteomatics-weather-map.rst:2]: WeatheR

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+		"time_series": [
+		  "2019-10-01T00:00:00+00:00/2019-10-03T23:59:59+00:00",
+		  "2018-10-01T00:00:00+00:00/2018-10-03T23:59:59+00:00"
+		],
+		"time_interval": 3
+	  }
+	}
+
+
+In this example, we used the ``time_series`` parameter and selected two specific time. The variable  ``prob_precip_1h:p`` was also added. In this example we query for each date range in 3 hour intervals for the 4 variables specified above. As described previously the output format is Geotiff.
```
Comment (thaisbendixen42) [inline source/up42-blocks/data/meteomatics-weather-map.rst:300]: "[...] and selected two specific time ranges. [...]"

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 4 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst]: Good catch!

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+.. warning::
+
+  In this block you can choose ``pixel_resolution`` to be a number from 8
+  to 3000. Based on this number and the selected AOI, **width** and **height** of the
+  final output image (in pixel) will be calculated.
+
+.. tip::
+
+  for checking the other possible ``colormap_style`` you can check the `Meteomatics website
```
Comment (rodrigoalmeida94) [inline source/up42-blocks/data/meteomatics-weather-map.rst:60]: Suggestion: The possible ``colormap_style`` options can be found in the `Meteomatics website`...

**Review by rodrigoalmeida94 — APPROVED** [review]
Nice! Just one suggestion

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 7]
```diff
@@ -0,0 +1,165 @@
+.. meta::
+   :description: UP42 data blocks: Weather/Ocean data forecasts block
+   :keywords: weather, ocean, forecast, meteomatics, model data,
+              observational data, time series
+
+.. _meteomatics-weather-map-block:
+
+Meteomatics - Weather/Ocean Data and Forecasts Map
+================================
```
Comment (perusio) [inline source/up42-blocks/data/meteomatics-weather-map.rst:9]: Make sure the underline to mark the header runs the full length of the title.

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+.. tip::
+
+  The possible ``colormap_style`` options can be found in the `Meteomatics website
+  <https://www.meteomatics.com/en/api/wms/#getcapabilitiesrequest>`_ where *STYLE* parameter
+  is described.
+
+.. warning::
+  Here is the list of variables that are **not** available via meteomatics WMS service:
+  is_slippery_road_3h:idx (also for 6h and 12h), power_line_oscillation_24h:idx, weather_code_1h:idx (also for 3, 6, 12, 24h),
```
Comment (perusio) [inline source/up42-blocks/data/meteomatics-weather-map.rst:66]: Can we put this into an itemized list please?

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+		"time_series": [
+		  "2019-10-01T00:00:00+00:00/2019-10-03T23:59:59+00:00",
+		  "2018-10-01T00:00:00+00:00/2018-10-03T23:59:59+00:00"
+		],
+		"time_interval": 3
+	  }
+	}
+
+
+In this example, we used the ``time_series`` parameter and selected two specific time ranges. The variable  ``prob_precip_1h:p`` was also added. In this example we query for each date range in 3 hour intervals for the 4 variables specified above. As described previously the output format is Geotiff.
```
Comment (perusio) [inline source/up42-blocks/data/meteomatics-weather-map.rst:164]: GeoTIFF.

File: source/up42-blocks/data/meteomatics-weather-map.rst [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+
+* ``bbox``: The bounding box to use as an area of interest (AOI). Will return all scenes that intersect with this box. Use only ``bbox``
+  **or** ``intersects``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use only ``bbox``
+  **or** ``intersects``.
+* ``time``: A date range to filter scenes on. This range applies to the acquisition date/time of the scenes.
+* ``time_series``: An array of date range filters as defined by ``time``. If defined, the ``limit`` parameter applies to each date range individually and the ``time`` filter is ignored.
+* ``time_interval``: A desired spacing in hours between the start and end point in time. By default, ``time_interval`` is set to 3 hours.
+* ``pixel_resolution``: A number in pixels per degree unit which define the final width and height (in pixel) of the final output. By default, ``pixel_resolution`` is set to 100.
+* ``colormap_style``: A single color scheme which should be used to represent the data. By default, ``colormap_style`` is set to JET.
```
Comment (perusio) [inline source/up42-blocks/data/meteomatics-weather-map.rst:41]: Can you please provide a list of possible values or a link to where the possible values are described for `colormap_style`?

**Review by perusio — CHANGES_REQUESTED** [review]
See below for comments. Thanks.

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 8 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:66]: Sure, I was lazy to do it to be honest :D But now that you require it, I will do!

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 10 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:41]: I added the link for possible color_style in the `tip` part.

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 7 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:9]: Will do

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 9 above)
Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst:164]: Added!

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 10 above)
Comment (perusio) [reply source/up42-blocks/data/meteomatics-weather-map.rst:41]: My bad 🙀

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 8 above)
Comment (perusio) [reply source/up42-blocks/data/meteomatics-weather-map.rst:66]: 💪

Comment (nekhtiari) [reply source/up42-blocks/data/meteomatics-weather-map.rst]: I added the table @perusio, please have a look and let me know if it is good.

**Review by perusio — COMMENTED** [review]
Nice. A table is always nice, although quite tricky to do in ReST. Thanks @nekhtiari 😸

File: source/up42-blocks/data/meteomatics-weather-map.rst (hunk 8 above)
Comment (perusio) [reply source/up42-blocks/data/meteomatics-weather-map.rst]: Thanks

**Review by perusio — APPROVED** [review]
LGTM.

**Pull request merged**